docs/lab/workflow/create-test.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: "Creating a Lab Test (Beta)"
+---
+
+Vital provides a way to create your own lab tests. These are made up of:
+
+1. One or more markers' ids.
+2. The lab id.
+3. The collection method (`testkit`, `walk_in_test`, `at_home_phlebotomy`).
+4. The sample type (`dried_blood_spot`, `serum`).
+
+Once a Lab test is created and approved, you can use it when making a order.
+
+<Warning>
+  When you create a Lab Test, it is not immediately available. We need to
+  validate and approve it on our side, before you can use it when making orders.
+</Warning>
+
+### Example
+
+To begin, make a request to our [labs endpoint](/api-reference/lab-testing/labs), to get all available labs.
+
+```json Get all available labs
+[
+  {
+    "id": 1,
+    "slug": "labcorp",
+    "name": "LabCorp",
+    "first_line_address": "123 Main St",
+    "city": "San Francisco",
+    "zipcode": "91789"
+  }
+]
+```
+
+Once you've picked a lab, you can fetch all available biomarkers. This is done via our [biomarkers endpoint](/api-reference/lab-testing/biomarkers).
+
+```json Get all biomarkers
+{
+  "markers": [
```

Comment:
Same question as before, the endpoint says biomarkers, but we are returning a list of markers.